### DISCUSSION TITLE
Multithreading in read_multi_vars

### DISCUSSION THREAD

**Discussion by Rahul14189** [gijzelaerr/python-snap7#459] [Q&A] [answered]
Does python snap7 methods "read_db" and "read_multi_vars", supports multithreading?  As i am trying to execute, it returns first thread result whereas for other threads, it raises error, stating "b'CLI : Job pending'" !! 
If not is there any other method that supports the same!

**Comment by spreeker** [comment, marked as answer]
No.

Have a single worker / queue executive commands to and from a plc. Response
time in minimum 5 Ms.

A plc is very slow compared to PC.  Makes not sense at all to do stuff
concurrently. Very dangerous to and debuggen would be no longer
deterministic.

Cheers

Stephan.

**Reply by Rahul14189** [reply]
Thanks @spreeker
